### DISCUSSION TITLE
No WebUI After Upgrade

### DISCUSSION THREAD

**Discussion by majorsl** [arendst/Tasmota#12052] [Q&A] [answered]
Hello all, hopefully this isn't in here already, if it was my search-fu failed me. I did an OTA upgrade of 9.4.0 on a RGB bulb that I've done a few times before. The upgrade started and, as usual with my Tasmota devices, I just left it alone to do its thing.  About 15 minutes later I noticed it had gone into AP mode with its normal name (strange), so I figured I'd just have to hit the WebUI and re-select my WiFi.  My phone attempts to connect to it but the UI times out.  I tried to do the quick power cycle recovery to no avail.

I'm hoping to salvage it without taking it apart (again) for a re-flash. Any ideas? It is probably the oldest one in my 75 device Tasmota fleet and only RGB (not even W or WW) so it may not be worth the hassle of disassembly.

Thanks for any input!

**Comment by barbudor** [comment, marked as answer]
If you can't reach the Web GUI on 192.168.4.1 from the `tasmota_XXXX` AP, it is probably bricked and only serial flashing can recover it.
The fact that it provides an AP with the original `tasmota_XXXX` name is pure coincidence (in many case it becomes `ESP_XXXX`) and means nothing.
Did you followed the upgrade path ?

**Reply by majorsl** [reply]
Thanks for the really fast reply. Yes, I followed the upgrade path directions, but in this case it was 9.1 -> 9.4.

It is actually still coming up with the name it was originally assigned by me: officelamp_xxxx so some part of the configuration was retained.  I note on reboot it does briefly show ESP_xxxx and then the above name. Maybe serial flash time.
